website/static/data/events.json

```diff
@@ -1,4 +1,8 @@
 [
+  {
```

Comment:
Could you also add weekly reports after line 5 from November as well?

https://apisix.apache.org/blog/2021/12/02/weekly-report-1130
https://apisix.apache.org/blog/2021/11/16/weekly-report-1114